app/scripts/lib/rpc-method-middleware/createMethodMiddleware.js

```diff
@@ -1,5 +1,8 @@
+///: BEGIN:ONLY_INCLUDE_IN(flask)
+import { handlers as permittedSnapMethods } from '@metamask/rpc-methods/dist/permitted';
+///: END:ONLY_INCLUDE_IN
 import { permissionRpcMethods } from '@metamask/snap-controllers';
-import { selectHooks } from '@metamask/rpc-methods';
+import { selectHooks } from '@metamask/rpc-methods/dist/utils';
```

Comment:
This change is not needed - we don't need to include the path to the `utils` module

Reply:
Fixed in: 255d6033e